Junchao-Mellanox/sonic-platform-daemons
# Support fan status monitor, temperature monitor and thermal policy control

What I did?
1. Add FanUpdater to get fan information and write it to database
2. Add TemperUpdater to get temperature information and write it to database
3. Add ThermalControlDaemon to run the thermal policy 

How to verify?
1. show platform fanstatus
2. show platform temperature
3. View syslog make sure thermalctld is up and there is no error logs
